corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, username=None, items=False):
         self.username = username
         self.items = items
         self.num_items = 0
+        self.finalized = False
```

Comment:
I like the idea of having a finalized variable. one thing i was always concerned about was if a payload got accidentally written to after it was finalized. maybe we could even though an error `__add__` or `append`  functions if someone tries to modify the response after it was modified